src/main/java/org/jitsi/videobridge/stats/PubSubStatsTransport.java

```diff
@@ -324,12 +324,15 @@ private void serviceChanged(ServiceEvent ev)
                 }
                 catch (IllegalArgumentException ex)
                 {
+                    logger.debug( "An unexpected exception occurred.", ex );
```

Comment:
Can you please collapse these three catch block into one with the java 7 "|" syntax?